### DISCUSSION TITLE
Does OpenEBS LocalPV HostPath support allowVolumeExpansion？

### DISCUSSION THREAD

**Discussion by jinteng123** [openebs/openebs#3672] [Q&A] [answered]
Does OpenEBS LocalPV HostPath support allowVolumeExpansion

**Comment by niladrih** [comment, marked as answer]
Hi @jinteng123. LocalPV Hostpath mounts a path on the host node. It does not set bounds on the 'volume'. In fact, there is no clearly defined 'volume' when it comes to hostpath volumes. The size of the disk/device/filesystem is the only real limit for hostpath volumes.

The localpv provisioner uses the https://github.com/kubernetes-sigs/sig-storage-lib-external-provisioner library. It is not a CSI driver. So it does not have interfaces for resize/explansion. It will be a lot of work to add this feature to the provisioner, and this is not in the project's scope.
